pkg/apis/kubedirector/v1beta1/kubedirectorcluster_types.go

```diff
@@ -224,12 +226,12 @@ const (
 
 // RoleStatus describes the component objects of a virtual cluster role.
 type RoleStatus struct {
-	Name                string             `json:"id"`
-	StatefulSet         string             `json:"statefulSet"`
-	Members             []MemberStatus     `json:"members"`
-	EncryptedSecretKeys map[string]string  `json:"encryptedSecretKeys,omitempty"`
-	UpgradingMembers    map[string]*string `json:"upgradingMembers,omitempty"`
-	RoleUpgradeStatus   RoleUpgradeStatus  `json:"roleUpgradeStatus,omitempty"`
+	Name                string            `json:"id"`
+	StatefulSet         string            `json:"statefulSet"`
+	Members             []MemberStatus    `json:"members"`
+	EncryptedSecretKeys map[string]string `json:"encryptedSecretKeys,omitempty"`
+	MembersUpgrading    int               `json:"membersUpgrading,omitempty"`
```

Comment:
The kdcluster members field is int32, so let's make this int32 also.

Reply:
(or just remove it entirely as mentioned below)

Reply:
Currently changed to `int32`